styles/banner.css

```diff
@@ -0,0 +1,68 @@
+.banner-container {
+    background-image: -moz-linear-gradient( 90deg, rgb(255,255,255) 0%, rgb(243,243,243) 79%, rgb(231,231,231) 100%);
+    background-image: -webkit-linear-gradient( 90deg, rgb(255,255,255) 0%, rgb(243,243,243) 79%, rgb(231,231,231) 100%);
+    background-image: -ms-linear-gradient( 90deg, rgb(255,255,255) 0%, rgb(243,243,243) 79%, rgb(231,231,231) 100%);
+    min-height: 400px;
+    width: 100%;
+    align-items: flex-end;
+    flex-direction: row;
+}
+
+.banner{
+    flex-direction: row;
+    align-items: flex-end;
+    flex-wrap: wrap;
+
+    margin: 52px 0 0 0;
+    box-sizing: border-box;
```

Comment:
Please review using of `box-sizing: border-box;` as it seems to be redundant in many places.